src/clone.d

```diff
@@ -920,7 +920,7 @@ extern (C++) FuncDeclaration buildPostBlit(StructDeclaration sd, Scope* sc)
 
     /* Build our own "postblit" which executes a
      */
-    if (a || (stc & STCdisable))
+    if ((a && a.dim) || (stc & STCdisable))
```

Comment:
Indeed, I tried to make the patch as small as possible to ease the review but the logic was a bit hard to understand. The question about the generation of `__xpostblit` even when the postblit is disabled is still unanswered though.